spec/support/example_app_helper.rb

```diff
@@ -0,0 +1,37 @@
+require 'fileutils'
+
+module ExampleAppHelper
+  def create_example_app
+    FileUtils.cd(tmp_path) do
+      `rails new #{app_name} -d postgresql --skip-action-mailer --skip-action-mailbox --skip-action-cable --skip-sprockets --skip-listen --skip-javascript --skip-turbolinks --skip-system-test --skip-test-unit --skip-bootsnap --skip-spring`
+    end
+  end
+
+  def remove_example_app
+    FileUtils.rm_rf(example_app_path)
+  end
+
+  def install_good_job
+    File.open("#{example_app_path}/Gemfile", 'a') do |f|
+      f.puts "gem 'good_job'"
+    end
+
+    FileUtils.cd(example_app_path) do
+      `rails g good_job:install`
+    end
+  end
+
+  def example_app_path
+    File.join(tmp_path, app_name)
+  end
+
+  def tmp_path
+    @_tmp_path ||= File.join(File.dirname(__FILE__), '..')
+  end
```

Comment:
I don't like that this method gets mixed into the example, but I realize it's necessary to both have the root temp path and the path that has the rails app. I wonder if this method could be done away with because the only time the root temp path is needed is in the initial `rails new` command, which could be turned into `example_app_path.join('..')` I think.

Reply:
We also need it to generate the `example_app_path` [here](https://github.com/bensheldon/good_job/pull/94/files#diff-cfad4f51e9f5a80e6676b6c3600f0de0R25) 🙂

Reply:
I'm going to resolve this and make a broader suggestion :-) Almost there!